src/backend/dx11/src/conv.rs

```diff
@@ -785,6 +785,7 @@ pub fn map_stage(stage: Stage) -> spirv::ExecutionModel {
         Stage::Compute => spirv::ExecutionModel::GlCompute,
         Stage::Hull => spirv::ExecutionModel::TessellationControl,
         Stage::Domain => spirv::ExecutionModel::TessellationEvaluation,
+        Stage::Task | Stage::Mesh => panic!("{:?} shader is not supported in DirectX 11", stage),
```

Comment:
I just realized we don't need `Stage` at all now. Filed https://github.com/gfx-rs/gfx/issues/3241 to pursue separately...

Reply:
Should I just remove It for now and wait until #3241 is resolved?

Reply:
only if you want to help resolving #3241. Otherwise, we'll remove this code later.

Reply:
I am fraid I don't have that much time currently :(. Have to work on my diploma thesis. So If that is alright, I will leave It for now.